rev_string.rb

```diff
@@ -0,0 +1,20 @@
+def rev_string
+  puts "enter a string...idiot"
+  org_string = gets.chomp
+
+  temp_arr = []
+  reverse_arr = []
+  i = org_string.length
+
+  temp_arr.push(org_string.split(""))
+
+  temp_arr.each do |char|
+    while i.length >= 0
```

Comment:
Not sure the use of a "while" loop inside an "each" loop is necessary but good workflow